packages/editor/styles/styles.css

```diff
@@ -530,30 +537,52 @@ p > *::selection {
   transform: rotate(-90deg);
 }
 
+[dir="rtl"].outline-list li.collapsed > p::before {
+  transform: rotate(90deg);
+}
+
 .outline-list li .outline-list::before {
   content: " ";
   position: absolute;
   height: 100%;
+  transition: border 200ms ease-in-out;
+}
+
+.outline-list:not([dir="rtl"]) li .outline-list::before {
```

Comment:
These `:not` css rules are not required. You can move the changed styles to the main CSS rule for the element.

The `[dir="rtl"]` should always be below the non-rtl rules. CSS Rules lower in the file take precedence.